common/lib/xmodule/xmodule/modulestore/__init__.py

```diff
@@ -154,6 +154,17 @@ def __init__(self, bulk_ops_record_type, **kwargs):
         self.records = defaultdict(bulk_ops_record_type)
 
 
+class NullSignalHandler(object):
+    """
+    A null handler that does nothing
+    """
+    def send(self, *args, **kwargs):
+        """
+        No-op
+        """
+        pass
+
+
```

Comment:
The other "real" SignalHandler is defined in `common/lib/xmodule/xmodule/modulestore/django.py`. It feels wrong to have this one separate from that one without good reason.